packages/x-charts-pro/src/ResponsiveChartContainerPro/ResponsiveChartContainerPro.tsx

```diff
@@ -5,21 +5,28 @@ import { ResponsiveChartContainerProps } from '@mui/x-charts/ResponsiveChartCont
 import { ResizableContainer, useResponsiveChartContainerProps } from '@mui/x-charts/internals';
 import { getReleaseInfo } from '../internals/utils/releaseInfo';
 import { ChartContainerPro } from '../ChartContainerPro';
+import { ZoomProps } from '../context/ZoomProvider';
+import { useResponsiveChartContainerProProps } from './useResponsiveChartContainerProProps';
 
-export interface ResponsiveChartContainerProProps extends ResponsiveChartContainerProps {}
+export interface ResponsiveChartContainerProProps
+  extends ResponsiveChartContainerProps,
+    ZoomProps {}
 
 const releaseInfo = getReleaseInfo();
 
 const ResponsiveChartContainerPro = React.forwardRef(function ResponsiveChartContainerPro(
   props: ResponsiveChartContainerProProps,
   ref,
 ) {
+  const { chartContainerProProps, baseProps } = useResponsiveChartContainerProProps(props);
   const { chartContainerProps, resizableChartContainerProps, hasIntrinsicSize } =
-    useResponsiveChartContainerProps(props, ref);
+    useResponsiveChartContainerProps(baseProps, ref);
```

Comment:
What about using the community hook inside the `useResponsiveChartContainerProProps` such that this `baseProps` is only internal